UCSLibraries/src/main/java/it/cnr/iit/xacml/wrappers/PolicyWrapper.java

```diff
@@ -75,34 +74,35 @@ public static PolicyWrapper build( String policy ) {
     @Override
     public List<Attribute> getAttributesForCondition( String conditionName ) {
         Reject.ifBlank( conditionName );
-
+        Reject.ifTrue( conditionName.length() > 20 );
```

Comment:
why 20?

Reply:
I've put a constant there in order to set up a maximum length for the string

Reply:
ok!